symmer/projection/utils.py

```diff
@@ -51,7 +51,7 @@ def update_eigenvalues(
     stabilizers.coeff_vec = (-1) ** np.count_nonzero(
         np.bitwise_and(
             reconstruction, 
-            basis.coeff_vec==-1
+            np.asarray(basis.coeff_vec)==-1
```

Comment:
We often find these sorts of bugs in Symmer, whereby something that _should_ be an array is not (maybe a list or tuple) and thus the behaviour is not as intended.

Reply:
Mmm this is true. I think what you have done here isnt' the best though... We should probably make the variable an array when basis.coeff_vec is set... Do you agree? We could maybe go through the whole repo and check anytime we allow people to input lists.